stdlib/public/stubs/OptionalBridgingHelper.mm

```diff
@@ -58,7 +75,7 @@ - (NSString*)description {
 static id getSentinelForDepth(unsigned depth) {
   // For unnested optionals, use NSNull.
   if (depth == 1)
-    return id_const_cast(kCFNull);
+    return SWIFT_LAZY_CONSTANT(id_const_cast([objc_getClass("NSNull") null]));
```

Comment:
This burns a tiny bit of memory, which makes me sad. We could possibly not cache it, I'm not sure how hot this code path is.

Reply:
The cache is probably a good idea. I don't think you need `id_const_cast` if you're going through Objective-C, by the way.

Reply:
This code path should only come up in cases that shouldn't be that common to begin with (bridging a nil Optional<T> to a non-nullable `id`, such as when turning a `[T?]` into NSArray), so it may be sufficiently cold that the cache isn't needed.